test/cqlpy/cassandra_tests/porting.py

```diff
@@ -13,14 +13,14 @@
 import collections
 import struct
 import time
-from util import unique_name
+from ..util import unique_name
 from contextlib import contextmanager
 from cassandra.protocol import SyntaxException, InvalidRequest
 from cassandra.protocol import ConfigurationException
 from cassandra.util import SortedSet, OrderedMapSerializedKey
 from cassandra.query import UNSET_VALUE
 
-import nodetool
+from ...cqlpy import nodetool
```

Comment:
Specifying the cqlpy here is not necessary, I checked and
```
from .. import nodetool
```
also works - and is much nicer (among other things, it will allow us to rename "cqlpy" once again if we want).